web/job_runs_controller.go

```diff
@@ -55,14 +55,40 @@ func (jrc *JobRunsController) Create(c *gin.Context) {
 	}
 }
 
+// Update marks the JobRun no longer pending, and resumes the Job's pipeline.
+func (jrc *JobRunsController) Update(c *gin.Context) {
+	id := c.Param("RunID")
```

Comment:
RunID and not id? This is all internal, based on the key you used in router, so it's not critical, but I would just stick with the convention of id. We're already in the JobRuns resource.

Reply:
Good catch, maybe from before I went with shallow routes.